libs/util/tsconfig.build.json

```diff
@@ -4,7 +4,8 @@
     "compilerOptions": {
         "baseUrl": null,
         "paths": null,
+        "incremental": true,
```

Comment:
There were issues with this setting in the past with the api-extractor not having stable ordering. Is this ok now?